.github/workflows/meson.yml

```diff
@@ -19,7 +19,8 @@ jobs:
             name: Windows MSVC Release,
             os: windows-latest,
             msvc: true,
-            pkgmanager: choco,
+            pkgmanager: msys,
```

Comment:
`pkgmanager=msys` is never used iinm and is technically not entirely correct as a combination of choco and msys is used